### DISCUSSION TITLE
Getting error whein i run and open my app first time after reload it automatically fix the error im using NEXT.js 13

### DISCUSSION THREAD

**Discussion by FaisalFasi** [vercel/next.js#59119] [Help] [answered]
### Summary

i get this below error when i run my default page under app/page.js More im using redux and im fetching data from server so i make it client side rendering page.

Error Message:
` :3000/_next/static/c…nks/app/page.js:480 Uncaught SyntaxError: Invalid or unexpected token
:3000/_next/static/c…s/app/layout.js:348 Uncaught SyntaxError: Invalid or unexpected token ` 

Please help me to fix it

i tried many things but error only goes if i dont use redux related functions i.e useDispatch but i need to use it so i can make a dispatch request

### Additional information

```js
Layout.js
import Navbar from "@/components/navbar/Navbar";
// import Footer from "@/components/footer/Footer";
import Setup from "@/components/setup/Setup";
import { StoreProvider } from "./StoreProvider";
import "./globals.css";

export const metadata = {
  title: "Create Next App",
  description: "Generated by create next app",
};
export default function RootLayout({ children }) {
  console.log("RootLayout");
  return (
    <html lang="en">
      <body suppressHydrationWarning={true}>
        <StoreProvider>
          <Setup />
          <Navbar />
          <div className="pt-[88px]">{children}</div>
        </StoreProvider>
      </body>
    </html>
  );
}

Default page.js under app folder 
"use client";
import React from "react";
import { useEffect } from "react";
import { useDispatch, useSelector } from "react-redux";
import { fetchJobs } from "@/lib/GlobalRedux/Features/jobs/JobsSlice";

import Card from "@/components/Card/Card";
import Search from "@/components/Search/Search";

const Home = () => {
  const dispatch = useDispatch();

  // Selectors
  console.log("Home");

  const jobs = useSelector((state) => state.jobs?.jobs);

  console.log("jobs", jobs);

  useEffect(() => {
    const fetchJobsData = () => {
      dispatch(fetchJobs());
    };
       console.log("fetchJobsData");
      fetchJobsData();
    
  }, [dispatch]);

  return (
    <div className="w-full p-8">
      <Search />
      Hello World
      <div>
        {jobs?.map((job, idx) => (
          <Card key={idx} job={job} />
        ))}
      </div>
    </div>
  );
};

export default Home;
```

**Comment by marcosmnz** [comment]
Hello @FaisalFasi, were you able to solve your error? 
I have this error too and apparently it's because of the version of Next, if you update to the latest version it disappears.

**Reply by FaisalFasi** [reply]
Hi, 
Yes i fixed it. You just need to upgrade your version to latest one. If it doesn't fix then try downgrade to the LTS version.

**Reply by harish-github35** [reply]
im using 14.1.0 and it is happening to me. @marcosmnz how did you fix it. should i use canary releases.

**Reply by FaisalFasi** [reply, marked as answer]
Hi,
Try to downgrade your project to  "next": "13.5.4", version.
hopefully it will fix the problem.

**Reply by harish-github35** [reply]
are you dumb? why you marking your workarounds as answer. it doesnt solve anything. why tf is this shit happening. somebody from next should look into this.

**Reply by FaisalFasi** [reply]
Dumb? I think you are out of your mind.
Be a little gentle and dont show me where you belongs too.Go and read the official documentation then you will get to know the problem and how to fix it.
For the problem I mentioned above has a fix i told you. Thats why i marked as solved. 
Go check your problem once again. Otherwise it's you who looks dumb.

**Reply by 1pharaxh** [reply]
is anyone able to fix it ? experiencing this on ubuntu, node 20 next 14

**Reply by dayojiboye** [reply]
Does anyone have a fix without downgrading?

**Reply by sjiep** [reply]
I actually got around this issue by using a [Template](https://nextjs.org/docs/app/api-reference/file-conventions/template) as the component that I included in the `Layout` wasn't entirely static.
